src/Model/BlogPost.php

```diff
@@ -312,6 +312,11 @@ public function getCMSFields()
                 );
             }
 
+            $shouldLazyLoadCategories = true;
+            if($this->Categories()->count() < 15) {
+                $shouldLazyLoadCategories = false;
+            }
```

Comment:
This can be changed to be a bit more readable (IMO) and also a bit better to phrase it in the positive sense "lazy load if more than 15" rather than currently "don't lazy load if less than 15".

```suggestion
            $shouldLazyLoadCategories = $this->Categories()->count() > 15;
```